Source/View Controllers/TCCProfileViewController.swift

```diff
@@ -159,7 +159,9 @@ class TCCProfileViewController: NSViewController {
             if response == .OK {
                 panel.urls.forEach {
                     guard let executable = self.model.loadExecutable(url: $0) else { return }
-                    block(executable)
+                    if self.model.selectedExecutables.firstIndex(of: executable) == nil {
```

Comment:
We could extract this into a small function and use it for both checks. It helps with clarity about what the code is doing and also helps if we change the logic later, we won't have to remember to change both places.

    func shouldExecutableBeAdded(_ executable: Executable) -> Bool {
        return self.model.selectedExecutables.firstIndex(of: executable) == nil
    }